### PR TITLE
Add helx-ui frontend

### PR DESCRIPTION
**Pull request by n0mn0m**
This adds serving the new react frontend to the appstore.

- Django app frontend added to serve the helx-ui assets
- Setup multistage build to copy the build artifacts from helx-ui
- Add bin/appstore command to pull frontend assets into the local environment
- Add docs to frontend/views.py describing how this works
- Update README

ref: helxplatform/development#668 helxplatform/development#643

### CODE REVIEW

File: appstore/appstore/urls.py [hunk 1]
```diff
@@ -35,10 +37,17 @@
 
     path('irods/login', app_core_views.IrodsLogin.as_view(), name='irods_login'),
```
Comment (ismagilani) [inline appstore/appstore/urls.py:38]: this can be removed since irods_login no longer exists

**Review by ismagilani — APPROVED** [review]

File: appstore/appstore/urls.py (hunk 1 above)
Comment (n0mn0m) [reply appstore/appstore/urls.py:38]: Can we remove all `irods` code, for instance `python-irodsclient==0.8.2 ` and the IRODS code in `core/views.py`?

Comment (ismagilani) [reply appstore/appstore/urls.py:38]: I don't see why not.

Comment (n0mn0m) [reply appstore/appstore/urls.py:38]: I'll include that in a future PR.

**Pull request merged**